docs/Examples/watch-only-wallet.js

```diff
@@ -0,0 +1,62 @@
+const {HDPrivateKey,Network} = require('bcoin');
+const {Mnemonic} = require('bcoin/lib/hd');
```

Comment:
Can't you combine these two lines? Pretty sure `Mnemonic` is available from root `bcoin` package.